### ISSUE TITLE
this qr code does not work in mobile browsers

### ISSUE THREAD

**Issue by m3tam0rph** [mebjas/html5-qrcode#813]
everything is ok on desktop browsers, but on mobile it doesn’t work on ios or android
![image](https://github.com/mebjas/html5-qrcode/assets/5280611/638b2ea7-9b52-41e8-841f-9f90e47e7b6d)

**Comment by t-hauck** [comment]
Reading qrcode works perfectly for me.
Notice that your image has a black background around the qrcode, and the qrcode also has black inside.
I think this really won't work.

Try an image with white borders (or another color) and black interior.
Run the test, and return the result here.

See this qrcode example that has a white background around it:
https://chart.apis.google.com/chart?chs=500x500&cht=qr&chl=https://github.com/mebjas/html5-qrcode/issues/813&choe=UTF-8&chld=H

**Comment by m3tam0rph** [comment]
> Reading qrcode works perfectly for me. Notice that your image has a black background around the qrcode, and the qrcode also has black inside. I think this really won't work.
> 
> Try an image with white borders (or another color) and black interior. Run the test, and return the result here.
> 
> See this qrcode example that has a white background around it: https://chart.apis.google.com/chart?chs=500x500&cht=qr&chl=https://github.com/mebjas/html5-qrcode/issues/813&choe=UTF-8&chld=H

yes your qr code can be scanned without problems. But unfortunately I can't change the qr code style.

**Comment by m3tam0rph** [comment]
i tried to invert the image with this qr code and it still doesn't work
![image (1)](https://github.com/mebjas/html5-qrcode/assets/5280611/b1ac50e1-49eb-4e3d-99bc-32ee75a9d41f)

**Comment by angela1393aa** [comment]
![image](https://github.com/mebjas/html5-qrcode/assets/79739270/dfb4ddf7-9846-429d-b3eb-35016bc94f72)
